packages/datagateway-search/src/searchPageTable.test.tsx

```diff
@@ -5,26 +5,33 @@ import { MemoryRouter } from 'react-router';
 
 import SearchPageTable from './searchPageTable';
 
-import { mount as enzymeMount } from 'enzyme';
-import { createMount } from '@material-ui/core/test-utils';
+import { mount as enzymeMount, shallow as enzymeShallow } from 'enzyme';
+import { createMount, createShallow } from '@material-ui/core/test-utils';
 import configureStore from 'redux-mock-store';
 import thunk from 'redux-thunk';
 import { initialState } from './state/reducers/dgsearch.reducer';
 import { dGCommonInitialState } from 'datagateway-common';
 import { setCurrentTab } from './state/actions/actions';
+import axios from 'axios';
 
 describe('SearchPageTable', () => {
   let mount: typeof enzymeMount;
+  let shallow: typeof enzymeShallow;
   let state: StateType;
 
   beforeEach(() => {
     mount = createMount();
+    shallow = createShallow({ untilSelector: 'Paper' });
```

Comment:
Is this shallow snapshot being rendered to the correct selector? As I'm unsure of its exact function, could use clarification from Louise/Victor.

Reply:
Can't really remember this one, but from looking at the snapshot for this test rendering until `Paper` stops the individual `InvestigationSearchTable`, `DatasetSearchTable`, `DatafileSearchTable` from rendering. Maybe these were causing some excess logging, or maybe it was just to make the snapshot smaller? If there was a reason to have them in the snapshot it could be reverted or just make sure it's being covered by the tests that actaully mount.

Reply:
Can't see any logging when changing it back to `mount` however the snapshot gets bigger as `InvestigationSearchTable`, `DatasetSearchTable`, `DatafileSearchTable` are rendered. Looking at the coverage, all lines are covered when using `shallow` so I would just leave it as it is.